mobile/src/components/course-modules/components/module.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC, ReactElement } from 'react';
+
+import { Text, View } from '~/components/common/common';
+
+import { styles } from './styles';
+
+type Props = {
+  index: number;
+  title: string;
+  description: string | null;
+};
+
+const Module: FC<Props> = ({ index, title, description }): ReactElement => {
+  return (
+    <View
+      style={{ ...(Boolean(index) && { marginTop: 15 }), ...styles.container }}
```

Comment:
Use Stack component for this
```suggestion
      style={styles.container}
```